src/integrationTest/java/uk/gov/hmcts/reform/juddata/cameltest/testsupport/IntegrationTestSupport.java

```diff
@@ -1,9 +1,26 @@
 package uk.gov.hmcts.reform.juddata.cameltest.testsupport;
 
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.sql.Timestamp;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
 import org.springframework.util.ResourceUtils;
+import uk.gov.hmcts.reform.juddata.camel.binder.JudicialOfficeAuthorisation;
 
 public interface IntegrationTestSupport {
 
+    static final String BLANK = "";
```

Comment:
don't need to define this...StringUtils has that use it.

Reply:
done